custom_components/tuya_local/devices/ecostrad_iq_heating_element.yaml

```diff
@@ -0,0 +1,116 @@
+name: Ecostrad iQ Heating Element
+primary_entity:
+  entity: climate
+  icon: "mdi:radiator"
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: true
+          value: "heat"
+        - dps_val: false
+          value: "off"
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: "auto"
+          value: Program
+          icon: "mdi:clock-outline"
+        - dps_val: "eco"
+          value: Eco
+          icon: "mdi:moon-waxing-crescent"
+        - dps_val: "hot"
+          value: Comfort
+          icon: "mdi:weather-sunny"
+        - dps_val: "only_inside"
+          value: Radiator
+          icon: "mdi:radiator"
+    - id: 16
+      type: integer
+      name: temperature
+      range:
+        min: 70
+        max: 300
+      mapping:
+        - scale: 10
+          step: 5
+    - id: 24
+      type: integer
+      name: current_temperature
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: number
+    name: Radiator Mode Temperature
+    category: config
+    dps:
+      - id: 57
+        type: integer
+        name: value
+        unit: C
+        range:
+          min: 300
+          max: 700
+        mapping:
+          - scale: 10
+            step: 50
+  - entity: lock
+    name: Child lock
+    category: config
+    dps:
+      - id: 40
+        type: boolean
+        name: lock
+        mapping:
+          - dps_val: true
+            icon: "mdi:hand-back-right-off"
+          - dps_val: false
+            icon: "mdi:hand-back-right"
+  - entity: select
+    name: Open window detection
+    category: config
+    dps:
+      - id: 108
+        type: string
+        name: option
+        mapping:
+          - dps_val: 0
+            value: "Off"
+            icon: "mdi:window-closed"
+          - dps_val: 60
+            value: "60 mins"
+            icon: "mdi:window-open"
+          - dps_val: 90
+            value: "90 mins"
+            icon: "mdi:window-open"
+  - entity: number
+    name: Boost time
+    category: config
+    dps:
+      - id: 111
+        type: integer
+        name: value
+        unit: hours
+        range:
+          min: 0
+          max: 7
+  - entity: number
+    name: Calibration offset
+    category: config
+    dps:
+      - id: 27
+        type: integer
+        name: value
+        unit: C

```

Comment:
Leave the units off these relative temperature settings, or make them general degrees (using the Unicode character for degree sign) because they do unexpected things if Home Assistant decides to apply unit conversion.